targets/targets.json

```diff
@@ -1552,13 +1575,17 @@
         "inherits": ["FAMILY_STM32"],
         "core": "Cortex-M0+",
         "extra_labels_add": ["STM32L0", "STM32L053x8", "STM32L053C8"],
-        "macros": ["RTC_LSI=1"],
         "config": {
             "clock_source": {
                 "help": "Mask value : USE_PLL_HSE_EXTC | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI",
                 "value": "USE_PLL_HSE_EXTC|USE_PLL_HSI",
                 "macro_name": "CLOCK_SOURCE"
-            }
+            },
+            "rtc_lsi": {
+                "help": "Use internal low speed clock (default clock is LSE)",
+                "value": "1",
+                "macro_name": "RTC_LSI"
```

Comment:
I made earlier this week a point about the macro name. what is the reason for this short name, however this is more specific macro related to LSI, this should be ok not to cause conflicts in the user space.

What are the other values accepted, 1 is LSE , 0 is ? 2 is ?